server.py

```diff
@@ -30,7 +30,20 @@ def create_application():
 SCHEMA = CollectJSONSchema()
 
 
-@APP.route("/api/v0/version", methods=['GET'])
+@APP.route("/r/insights/platform/aiops-data-collector", methods=['GET'])
```

Comment:
Can we please define this route prefix as a global variable? It's much better than repeat it every time and possibly cause typos...

```python
ROUTE_PREFIX = "/r/insights/platform/aiops-data-collector"
...
@APP.route(f'{ROUTE_PREFIX}', method=['get'])
...
@APP.route(f'{ROUTE_PREFIX}/api/v0/version', method=['get'])
```

If we can do the same in the Swagger YAML, that would be also nice.

Reply:
Good point - thanks!